package.json

```diff
@@ -19,7 +19,9 @@
     "coverage": "yarn test --coverage",
     "prepare": "husky install",
     "extract-translations": "i18next 'src/**/*.component.tsx' 'src/**/*.tsx' --config ./i18next-parser.config.js",
-    "postinstall": "husky"
+    "postinstall": "husky",
+    "test-e2e": "playwright test",
+    "playwright": "playwright test"
```

Comment:
Are these two scripts being used?

```code
   "test-e2e": "playwright test",
   "playwright": "playwright test"

Reply:
Both scripts are essentially doing the same thing, have removed one

Reply:
The two scripts are still existing, could you double check?